### PR TITLE
Benchmark model checker with cargo bench

### PR DESCRIPTION
**Pull request by jenrik**
Use `cargo bench` to benchmark the model checker om various test cases. Requires the nightly compiler because the `test` feature is not yet stable.

Feel free to contribute test cases by committing to this branch.

### CODE REVIEW

File: src/edg.rs [hunk 1]
```diff
@@ -1072,4 +1079,23 @@ mod test {
...
+    #[bench]
+    fn bench_mexican_standoff(b: &mut Bencher) {
+        b.iter(|| {
+            let json = String::from_utf8_lossy(include_bytes!("../mexican-standoff.json"));
+            let game_structure: EagerGameStructure = serde_json::from_str(json.to_str()).unwrap();
+            let formula = Arc::new(Phi::Next {
+                players: vec![0],
+                formula: Arc::new(Phi::Proposition(0)),
+            });
+            let graph = ATLDependencyGraph { game_structure };
```
Comment (NicEastvillage) [inline src/edg.rs:1092]: We might want to move this out of `b.iter` to avoid measure IO and setup.

Comment (jenrik) [reply src/edg.rs:1212]: agreed, I just had some trouble with ownership and didn't bother finding a solution

Comment (NicEastvillage) [reply src/edg.rs:1212]: I tried too. Same conclusion xD

Comment (NicEastvillage) [reply src/edg.rs:1212]: I tried making the exact same closure in a #[test] and it worked perfectly, so I think it is a problem with how #[bench] modifies the function. I suggest moving to Criterion.rs

Comment (cogitantium) [reply src/edg.rs:1212]: Agreed, I'm working on setting up criterion - so far it looks pretty slick:
```
Benchmarking fib 20
Benchmarking fib 20: Warming up for 3.0000 s
Benchmarking fib 20: Collecting 100 samples in estimated 5.0856 s (207k iterations)
Benchmarking fib 20: Analyzing
fib 20                  time:   [25.321 us 25.613 us 25.959 us]
Found 15 outliers among 100 measurements (15.00%)
  3 (3.00%) high mild
  12 (12.00%) high severe
```

**Comment by jenrik** [comment]
Superseded by #64

**Pull request closed without merging**